service/config_provider.go

```diff
@@ -79,8 +81,13 @@ type configProvider struct {
 // * Then unmarshalls the final config.Config using the given configunmarshaler.ConfigUnmarshaler.
 //
 // Notice: This API is experimental.
-func NewConfigProvider(configMapProviders []configmapprovider.Provider, cfgMapConverters []ConfigMapConverterFunc, configUnmarshaler configunmarshaler.ConfigUnmarshaler) ConfigProvider {
+func NewConfigProvider(
+	selectors []string,
+	configMapProviders map[string]configmapprovider.Provider,
```

Comment:
>selectors []string,

Please document this parameter.

>configMapProviders map[string]configmapprovider.Provider,

What is the key of this map?

Reply:
The "scheme"